infrastructure/variables.tf

```diff
@@ -8,6 +8,11 @@ variable "location" {
   default = "UK South"
 }
```

Comment:
worth renaming this to `location_app` (or similar) now.

Reply:
Right. `location` and `location_db` is confusing. Will change.